bundle/Resources/views/ezadminui/field/edit/eztags.html.twig

```diff
@@ -27,13 +28,16 @@
             </div>
 
             {% set edit_view_template = null %}
-            {% for edit_view_config in ezpublish.configResolver.parameter('edit_views', 'eztags') %}
-                {% if edit_view_config.identifier == edit_view %}
-                    {% set edit_view_template = edit_view_config.template %}
+
+            {% for view_config in ezpublish.configResolver.parameter('edit_views', 'eztags') %}
```

Comment:
Can you bring back the old name `edit_view_config` now, to keep the diff smaller?